content/tutorials/jwt-authentication.textile

```diff
@@ -54,123 +78,156 @@ h2(#live-demo). Ably JWT auth example
     </div>
 </section>
 
-If you wish to directly check out the source code for this tutorial, we recommend you clone the tutorial repo and run it locally by following the steps below:
+If the server returns a valid JWT the client is authenticated with Ably, and you see an alert box with the message: "Client successfully connected to Ably using JWT auth".
 
-bc[sh]. git clone https://github.com/ably/tutorials.git
+The full code for this project can be found in its "Ably GitHub repository":https://github.com/ably/jwt-auth-node-tutorial.
 
-bc[sh]. git checkout jwt-authentication-nodejs
+h2(#running-locally). Running the project locally
 
-bc[sh]. cd jwt-authentication-node
+The project can be run locally using the following steps:
```

Comment:
I think this is great to split into its own section, but might be good to note this is the "quick" route, or the alternative is to build it step-by-step. An "in this step" like the other ones maybe?

Reply:
Good point, I've now stated what the section does, and provided a link to the step-by-step process too as an alternative.